packages/utils/src/structs/checkpoint.cairo

```diff
@@ -171,7 +171,7 @@ const _2_POW_184: felt252 = 0x10000000000000000000000000000000000000000000000;
 /// - `key` is stored at range [4,67] bits (0-indexed), taking the most significant usable bits.
 /// - `value.low` is stored at range [124, 251], taking the less significant bits (at the end).
 /// - `value.high` is stored as the second tuple element.
-impl CheckpointStorePacking of StorePacking<Checkpoint, (felt252, felt252)> {
+pub impl CheckpointStorePacking of StorePacking<Checkpoint, (felt252, felt252)> {
```

Comment:
```suggestion
impl CheckpointStorePacking of StorePacking<Checkpoint, (felt252, felt252)> {
```

This doesn't have to be public, since every time Checkpoint is imported, this implementation will be automatically in scope.

Reply:
Since the tests got moved to a new file I think this now needs to be `pub` for it to work. If we want to restrict the visibility we can make it public just inside the crate.
`pub(crate) impl CheckpointStorePacking of StorePacking<Checkpoint, (felt252, felt252)> {`

I'm taking this route unless you have a better suggestion(or we want to move the tests back to this file)

Reply:
We can remove the pub(crate) and test will still work if we change the impl import CheckpointStorePacking for the trait import StorePacking, and use StorePacking instead. Note that you need to explicitly declare the type of checkpoint in line 100 of the test file, but nothing else.

Reply:
Implementations usually don't need to be imported as long as the module and the appropriate trait are in scope.